src/modules/common/components/bars/sideBar/autoSignOut/warning.js

```diff
@@ -1,76 +0,0 @@
-/* istanbul ignore file */
-import React, { useState } from 'react';
-import Countdown from 'react-countdown';
```

Comment:
@ManuGowda `react-countdown` should be removed as it's not used anywhere else in the app